grpc/server/src/connection_initiator.h

```diff
@@ -0,0 +1,82 @@
+#include <future>
+
+#include "connection_result.h"
+#include "log.h"
+
+namespace dronecore {
+namespace backend {
+
+template <typename DroneCore>
+class ConnectionInitiator
+{
+public:
+    ConnectionInitiator() {}
+    ~ConnectionInitiator() {}
+
+    bool start(DroneCore &dc, const int port)
```

Comment:
What about connection URL ?

Reply:
Some further enhancement as well :-). Shouldn't be difficult to add, but not part of the "minimum viable product".